sdk/python/feast/permissions/client/auth_client_manager.py

```diff
@@ -44,3 +46,10 @@ def create_grpc_auth_header(auth_config: AuthConfig) -> Tuple[Tuple[str, str]]:
     token = auth_client_manager.get_token()
 
     return (("authorization", "Bearer " + token),)
+
+
+def create_flight_call_options(auth_config: AuthConfig) -> fl.FlightCallOptions:
+    if auth_config.type != AuthType.NONE.value:
```

Comment:
I feel like `create_flight_call_options` and `create_grpc_auth_header` methods does not belong to this file. How about moving them to respective implementations? or having separate util for auth client?

Reply:
can we do this next PR.  The reason @tmihalac  probably will change the function logic related to [PR](https://github.com/RHEcosystemAppEng/feast/pull/42) 
else @tmihalac if possible you can address this you can move these function by creating util file under permissions/client

Reply:
@redhatHameed ok

Reply:
@tmihalac Thanks